src/lib/cli.js

```diff
@@ -28,6 +28,7 @@ program
   .option('-d, --devMode', 'Shortcut for running cucumber-js directly')
   .option('--fail-fast', 'abort the run on first failure')
   .option('--strict', 'Fail fast if a step is undefined')
+  .option('--mergedLog', 'Path, relative to logdir, merged log. Specify "" to not generate one.')
```

Comment:
Also mention the default value in the command line help. Bonus points if you actually format the default value in to the help to keep it DRY.